schema/schema.go

```diff
@@ -10,144 +10,140 @@ type scoreSchema struct {
 	Value string `jsonschema:"value;description:eg. 3.4/4.0 - [Add obtained score/total score]"`
 }
 
+type resourceSchema struct {
+	URL   string `jsonschema:"url"`
+	Label string `jsonschema:"label"`
+}
+
 type coreSchema struct {
-	Name              string         `jsonschema:"name;description:e.g. John Doe"`
-	Title             string         `jsonschema:"title;description:e.g. Software Engineer"`
-	Image             string         `jsonschema:"image;description:e.g. example.com/Abcxyz - [URL (as per RFC 3986) to a image in JPEG or PNG format]"`
-	Email             string         `jsonschema:"email;description:e.g. lucas@example.com;format:email"`
-	Phone             string         `jsonschema:"phone;description:e.g. 912-217-7923 - [Phone numbers are stored as strings so use any format you like]"`
-	URL               string         `jsonschema:"url;description:e.g. http://www.example.com/my-slides/;format:uri"`
+	Name              string         `jsonschema:"name"`
+	Title             string         `jsonschema:"title"`
+	Image             string         `jsonschema:"image;description:URL (as per RFC 3986) to a image in JPEG or PNG format"`
+	Email             string         `jsonschema:"email"`
+	Phone             string         `jsonschema:"phone;description:hone numbers are stored as strings so use any format you like"`
```

Comment:
``` suggestion
	Phone             string         `jsonschema:"phone;description:Phone number"`
```

Reply:
Fixed in 570748d476f068d3be54b49ed54ae44398a2eeaa